app/src/main/java/org/mozilla/fenix/tabstray/browser/InactiveTabViewHolder.kt

```diff
@@ -37,7 +37,7 @@ sealed class InactiveTabViewHolder(itemView: View) : RecyclerView.ViewHolder(ite
 
         init {
             itemView.apply {
-                isActivated = InactiveTabsState.isExpanded
+                isActivated = itemView.context.components.appStore.state.inactiveTabsExpanded
```

Comment:
The actions are processed asynchronously using `lib-state`. Usually, we'd want all consumers to observer the store (incl. it's initial state). Is this good enough here to simply read directly?

Reply:
In this case, I think it's fine to just take the state. In our worst case, the tabs tray is not expanded the first time a user re-enters the tabstray if we haven't processed the async action, but it will in the following case.